themes/mongodb/src/widgets/deluge/deluge.js

```diff
@@ -99,15 +102,24 @@ class Deluge extends preact.Component {
         };
     }
 
+    validateFormLength(input) {
+        const hasError = !(input === '' || input.length >= MIN_CHAR_COUNT);
```

Comment:
Does this allow them to submit an empty comment?

Reply:
Yup, that's correct. We don't require a response to the freeform question—but if they do decide to answer, we want at least 15 characters of info.